frontend/src/container/OrganizationSettings/AuthDomains/index.tsx

```diff
@@ -254,39 +254,41 @@ function AuthDomains(): JSX.Element {
 
 	if (!isLoading && data?.payload?.length === 0) {
 		return (
-			<Space direction="vertical" size="middle">
-				<AddDomain refetch={refetch} />
+			<div data-testid="authenticated-domains-section">
```

Comment:
do we need extra `div` here ? we can add data-test-id to Space itself if thats the only usecase ?